homeassistant/components/sonos/sensor.py

```diff
@@ -64,3 +78,33 @@ def native_value(self) -> int | None:
     def available(self) -> bool:
         """Return whether this device is available."""
         return self.speaker.available and self.speaker.power_source
+
+
+class SonosAudioInputFormatSensorEntity(SonosEntity, SensorEntity):
+    """Representation of a Sonos audio import format sensor entity."""
+
+    _attr_entity_category = ENTITY_CATEGORY_DIAGNOSTIC
+    _attr_icon = "mdi:import"
+    _attr_should_poll = True
+
+    @property
+    def unique_id(self) -> str:
+        """Return the unique ID of the sensor."""
+        return f"{self.soco.uid}-audio-format"
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        return f"{self.speaker.zone_name} Audio Input Format"
+
+    @property
+    def native_value(self) -> str:
+        """Return the current state of the text sensor."""
+        return self.speaker.soundbar_audio_input_format
```

Comment:
You could probably use `_attr_*`'s instead.

Reply:
Will need an `__init__`. Can do that for all entity types as a followup. 👍

Reply:
Actually, this would clean things up a bit. Will update shortly.

Reply:
Done.